core/src/main/scala/scalafix/util/OrganizeImports.scala

```diff
@@ -200,14 +199,13 @@ private[this] class OrganizeImports private (implicit ctx: RewriteCtx) {
       val oldImports = getGlobalImports(code)
       val globalImports = oldImports.flatMap(getCanonicalImports)
       val cleanedUpImports = cleanUpImports(globalImports, patches)
-      val tokens = code.tokens
       val tokenToEdit =
         oldImports.headOption
           .map(_.tokens.head)
-          .getOrElse(tokens.head)
+          .getOrElse(ctx.tokens.head)
```

Comment:
The tree nodes that return empty tokens are incredibly few (if any) and I would say it was a truly exceptional case if those were passed in here. Top level trees always have a beginning of file token.

There's an interesting ongoing discussion in scalameta about validating trees for a given dialect. I would be interested to start a similar discussion about the possibility to use a non empty list type Where applicable. The challenge is try and encode as much as possible on the type level without burdening syntax too heavily. Quasi quotes help a lot here.